src/components/Dashboard/Form/Applicants/Applicants.js

```diff
@@ -1,66 +1,91 @@
 import React from "react";
 import PropTypes from "prop-types";
 
+import { useFormState } from "react-final-form";
+
 import FieldWrapper from "../FieldWrapper/FieldWrapper";
 import styles from "./Applicants.scss";
 
-const Applicants = ({ reasonsList, disabilityList }) => (
-  <div className={`FormContent ${styles.Applicants}`}>
-    <FieldWrapper
-      name={`applicant.names`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.names`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.city`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.postalCode`}
-      componentSize="medium"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.street`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.houseNumber`}
-      componentSize="medium"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.issue`}
-      componentSize="large"
-      component="select"
-      options={reasonsList}
-    />
-    <FieldWrapper
-      name={`applicant.reason`}
-      componentSize="extraLarge"
-      component="select"
-      options={disabilityList}
-    />
-    <FieldWrapper
-      name={`applicant.secondReason`}
-      componentSize="extraLarge"
-      component="select"
-      options={disabilityList}
-    />
-    <FieldWrapper
-      name={`applicant.period`}
-      componentSize="medium"
-      component="input"
-    />
-  </div>
-);
+const Applicants = ({ reasonsList, disabilityList }) => {
+  /**
+   * Calculate if second disable select is enabled
+   * Select should be enabled if - user want decision of special education
+   * and as a reason choose disability that can be engage with other
+   * @returns {bool}
+   */
+  const getSecondReasonDisabled = () => {
+    const actualValues = useFormState().values;
```

Comment:
asking just in case 'cause I haven't checked it if it works but documentations says `useFormState() is used internally inside <FormSpy/>.` but we don't have it :D

Reply:
In some magical way it works!